table/src/main/java/tech/ydb/table/description/TableIndex.java

```diff
@@ -6,13 +6,18 @@
 
 /**
  * @author Sergey Polovko
+ * @author Kirill Kurdyukov
  */
 public class TableIndex {
 
     public enum Type {
         GLOBAL,
     }
 
+    public enum ConsistencyType {
```

Comment:
If we look at https://github.com/ydb-platform/ydb-api-protos/blob/master/protos/ydb_table.proto#L63-L66, we can see, that global index and global async index are two different types for table index. They aren't two kind of one type. I think that instead of adding a new enum we can add new item of Type GLOBAL_ASYNC